src/Helper/Command/EmitOrderShippedEventCommand.php

```diff
@@ -0,0 +1,31 @@
+<?php declare(strict_types=1);
+/**
+ * This File is part of JTL-Software
+ *
+ * User: avermeulen
+ * Date: 2020-03-24
+ */
+
+namespace JTL\SCX\Lib\Channel\Helper\Command;
+
+use JTL\SCX\Client\Channel\Event\EventType;
+use Symfony\Component\Console\Input\InputOption;
+
+class EmitOrderShippedEventCommand extends AbstractEmitEventCommand
+{
+    protected static $defaultName = 'helper:emit.OrderShippedEvent';
+
+    protected function configure(): void
+    {
+        parent::configure();
+        $this->setDescription('Helper command to emit OrderShippedEvent for Testing')
+            ->addOption('orderId', null, InputOption::VALUE_OPTIONAL, 'orderId', null)
```

Comment:
use a VALUE_REQUIRED instead of VALUE_OPTIONAL otherwise it would be possible to use this like
`--orderId`

Remember a option is always optional ;)